Makefile

```diff
@@ -291,6 +291,7 @@ server: $(server_TARGET)
 
 # Patches binary headers to work around a mono bug
 fixheader.exe: packaging/fixheader.cs
+	@command -v $(CSC) || (echo "Mono is not installed. Please install Mono from http://www.mono-project.com/download/ before building OpenRA."; exit 1)
```

Comment:
Blah, I was doing this previously but removed it because I thought you were implying you wanted the output.